<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3DS Rich Presence</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="bot">
    <widget class="QLabel" name="description">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Add the bot's FC to your friend list before continuing to the next page.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="section">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="continueButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QLabel" name="botFCLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>XXXX-XXXX-XXXX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Link a Friend Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="description_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>420</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please be aware that by using this product you are agreeing to the &lt;a href=&quot;https://github.com/MCMi460/3DS-RPC/blob/main/TERMS.md&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Terms and Services&lt;/span&gt;&lt;/a&gt; listed on the MCMi460/3DS-RPC Github page.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="link">
    <widget class="QLabel" name="description2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Make certain that you have added the bot account's FC to your friendlist before pressing &quot;Log In&quot;.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="section2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fcInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>341</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter your friend code here!</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="title2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Link a Friend Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="description_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>420</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please be aware that by using this product you are agreeing to the &lt;a href=&quot;https://github.com/MCMi460/3DS-RPC/blob/main/TERMS.md&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Terms and Services&lt;/span&gt;&lt;/a&gt; listed on the MCMi460/3DS-RPC Github page.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main">
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="friendCard">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>451</width>
       <height>231</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="namePlate">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="username">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="miiLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>570</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>X Close</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gamePlate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="gameIcon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="gameName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>563</y>
       <width>601</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showElapsedOff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showElapsedOn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showProfileButtonOff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showProfileButtonOn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showSmallImageOff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showSmallImageOn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>370</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QLabel" name="showElapsedText">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow others to see your current time in-game?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="showProfileButtonText">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>170</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow others to click your status to view your profile?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="showSmallImageText">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>320</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show a small image of your Mii to Discord?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="showElapsed">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>65</y>
       <width>550</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="okButtonLogout">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>470</y>
       <width>270</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="showProfileButton">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>215</y>
       <width>550</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="showSmallImage">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>365</y>
       <width>550</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <zorder>showSmallImage</zorder>
    <zorder>showProfileButton</zorder>
    <zorder>showElapsed</zorder>
    <zorder>okButton</zorder>
    <zorder>showElapsedOff</zorder>
    <zorder>showElapsedOn</zorder>
    <zorder>showProfileButtonOff</zorder>
    <zorder>showProfileButtonOn</zorder>
    <zorder>showSmallImageOff</zorder>
    <zorder>showSmallImageOn</zorder>
    <zorder>showElapsedText</zorder>
    <zorder>showProfileButtonText</zorder>
    <zorder>showSmallImageText</zorder>
    <zorder>okButtonLogout</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
